src/stores/activitiesStores/Finalize.js

```diff
@@ -0,0 +1,77 @@
+import { observable, action, runInAction, reaction } from 'mobx';
+import _ from 'lodash';
+import { Token, OracleStatus, AppLocation, SortBy } from '../../constants';
+import { queryAllOracles } from '../../network/graphQuery';
+import Oracle from '../models/Oracle';
+
+const INIT = {
+  loaded: false, // loading state?
+  loadingMore: false, // for laoding icon?
+  list: [], // data list
+  hasMore: true, // has more data to fetch?
+  offset: 0, // skip
```

Comment:
why not skip?

Reply:
in NoSQL, `offset` and `limit` are conventions.

Reply:
oh sorry, just check docs, the new convention is `skip` and `limit`, will update it.